### PR TITLE
#PICA-258 setting up democracy proposal deposit as dicussed

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
for https://app.clickup.com/t/20465559/PICA-258

Increased amount of PICA to create proposal as measure against spam.

Also moved price per by of proposal to preimage config from democracy as it is belongs too.

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [PICA-258 Democracy proposals lock more tokens per proposal (ED from spam and security(](https://app.clickup.com/t/20465559/PICA-258)

**Review by andyjsbell — APPROVED** [review]
Thanks @dzmitry-lahoda

**Review by andyjsbell — APPROVED** [review]
thanks @dzmitry-lahoda

**Review by JafarAz — APPROVED** [review]

File: code/parachain/runtime/composable/src/lib.rs [hunk 1]
```diff
@@ -577,16 +577,15 @@ impl scheduler::Config for Runtime {
...
 }
 
 impl preimage::Config for Runtime {
 	type WeightInfo = preimage::weights::SubstrateWeight<Runtime>;
 	type RuntimeEvent = RuntimeEvent;
 	type Currency = Balances;
 	type ManagerOrigin = EnsureRoot<AccountId>;
-	type BaseDeposit = PreimageBaseDeposit;
-	type ByteDeposit = PreimageByteDeposit;
+	type BaseDeposit = ConstU128<100_000_000_000_000>;
```
Comment (blasrodri) [inline code/parachain/runtime/composable/src/lib.rs:587]: where are these numbers coming from?

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**